Makefile

```diff
@@ -69,8 +69,8 @@ endif
 # weird reports at golangci-lint step
 ifneq ($(HAS_SERVER),)
 	@echo Running golangci-lint
-	$(GO) vet ./...
-	$(GOBIN)/golangci-lint run ./...
+	$(GO) vet ./server/...
```

Comment:
Same concern from another PR where we are ignoring the `build` folder by doing this